test/controllers/csv_controller_test.rb

```diff
@@ -0,0 +1,198 @@
+require_relative '../test_helper'
+
+describe CsvController do
```

Comment:
This is currently set to allow anyone with access to generate and download a report.  Not sure what level of permissions is necessary.  

There may be use cases where a viewer should be able to get the report, so they can review the deploys data without being able to deploy.